lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -47,6 +47,13 @@
     CcxMembershipFactory,
     CcxFutureMembershipFactory,
 )
+from django.conf import settings
+
+FEATURES = settings.FEATURES.copy()
+FEATURES_WITHOUT_CUSTOM_GRADING = FEATURES.copy()
+FEATURES_WITHOUT_CUSTOM_GRADING['ENABLE_CUSTOM_GRADING'] = False
+FEATURES_WITH_CUSTOM_GRADING = FEATURES.copy()
+FEATURES_WITH_CUSTOM_GRADING['ENABLE_CUSTOM_GRADING'] = True
```

Comment:
It looks that you do not need FEATURES at all, 

FEATURES_WITH_CUSTOM_GRADING = settings.FEATURES.copy()
FEATURES_WITHOUT_CUSTOM_GRADING = settings.FEATURES.copy()

FEATURES_WITH_CUSTOM_GRADING['ENABLE_CUSTOM_GRADING'] = True
FEATURES_WITHOUT_CUSTOM_GRADING['ENABLE_CUSTOM_GRADING'] = False